index.html

```diff
@@ -636,6 +636,526 @@ <h3>Criteria and Criterion</h3>
+      </section> <!-- Criteria.Version -->
+      <section>
+        <h4>Criteria.Question</h4>
+        <p>This subcomponent presents the key question for evaluating the 
+        criteria. The question MUST be a single inquiry that gets to
+        the heart of the criteria. It should be reasonably clear to a 
+        competent professional skilled in the art how to resolve
+        the question into one of the possible responses.</p>
+      </section> <!-- Criteria.Question -->
+      <section>
+        <h4>Criteria.Responses</h4>
+        <p>This subcomponent lists the expected responses to the question.</p>
+        <ol>
+        <li>
+        Responses are either labeled or open ended.
+        </li>
+        <li>
+        Labeled responses MUST include a label and a meaning for that 
+        label.
+        </li>
+        <li>
+        Labels for a response MUST start with “A” in each criteria and progress sequentially through the English alphabet.
+        </li>
+        <li>
+        Open ended responses specify how an evaluator should fill in the response.
+        </li>
+        </ol>
+      </section> <!-- Criteria.Responses -->
+      <section>
+        <h4>Criteria.Relevance</h4>
+        <p>This subcomponent explains why this criteria is useful. Readers 
+        should be able to understand the extent to which this
+        particular criteria is applicable to their situation. 
+        </p>
+      </section> <!-- Criteria.Relevance -->
+      <section>
+        <h4>Criteria.Examples</h4>
+        <p>This section lists example evaluations of different DID methods 
+        using this criteria, presented as a table for easy
+        correlation between evaluations of different methods using the 
+        same criteria.
+        </p>
+        <ol>
+        <li>Each example must have the following entries, each its own column(s) in the examples table.
+        <ol type="a">
+          <li>Method</li>
+          <li>Responses (on column for each element)</li>
+          <li>Notes</li>
+        </ol>
+        </li>
+        <li>
+        Some criteria MAY require an additional "Referent" entry, in its
+        own column in the examples table.
+        </li>
+        <li>Example Evaluations for a given criteria SHOULD highlight 
+        distinct responses. That is, each evaluation should have a
+        different set of responses from the other example evaluations. 
+        The point of the examples is to illustrate how different
+        DID methods score differently.</li>
+        <li>The editors will curate the list of proposed examples to 
+        provide a best effort illustration of each criteria, with
+        consideration to highlighting a broad corpus of methods.</li>
+        <li>Criteria with fewer than 3 example evaluations MUST be 
+        marked “PROVISIONAL”</li>
+        <li>Criteria SHOULD have no more example evaluations than there 
+        are possible responses.</li>
+        </ol>
```

Comment:
I'm realizing that several of my suggestions above may not be relevant, as I'm becoming convinced that I've misunderstood the write-up, particularly of the `question` and `responses` and how they play out in the `evaluation`.  I was particularly thrown off by the `Responses (on column for each element)` (what's an element, now?  I do now see this term is used later, but it really should be defined before its first necessary use, which this does seem to be) which suggested multiple *columns* of `responses`, where it seems that there really should be *one* `response` column with one example *row* for each possible response...  

So I'll not be reviewing this draft further, until my suggestions above are addressed, which will hopefully clarify the rest.

Reply:
You caught it just as you diverged. All of the other suggestions I've adopted without change.

Perhaps a section that lays out the structure of the example section would serve to define the different parts.

Reply:
Also, queued up in #58 as a new issue